qcodes/instrument_drivers/rigol/DS4034.py

```diff
@@ -0,0 +1,106 @@
+"""
+The QCodes driver of the oscilloscope Rigol DS4034. Please see
+
+http://int.rigol.com/File/TechDoc/20160831/DS4000E_ProgrammingGuide_EN.pdf
+
+for the programming manual for this device
+"""
+
+import numpy as np
+
+from qcodes import VisaInstrument
+from qcodes.utils.validators import Ints, Bool
+
+
+class Rigol_DS4035(VisaInstrument):
+    def __init__(self, name, address, **kwargs):
+        super().__init__(name, address, terminator='\n', **kwargs)
+
+        self.add_parameter(
+            "data_format",
+            get_cmd=":wav:form?",
+            set_cmd=":wav:form {}",
+            val_mapping={
+                'asc': 'ascii',
```

Comment:
Is there a reason for these two strings to be abbreviated? Personally, I like long, verbose, telling values for string parameter.